torchacc/utils/consolidate_and_reshard_ckpts.py

```diff
@@ -0,0 +1,73 @@
+from argparse import ArgumentParser
+from torchacc.dist.state_dict_utils import consolidate_and_reshard_model_dict, consolidate_and_reshard_optim_dict
+
+MODEL_NAME_PATTERN = "rank*-of-*-model.pth"
+OPTIM_NAME_PATTERN = "rank*-of-*-optimizer.pth"
+
+
+def main():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--ckpt_dir",
+        type=str,
+        required=True,
+        help=(
+            f"The name pattern of the XLA FSDP checkpoint files to be consolidated. "
+            f"Files matching the pattern ``ckpt_dir + ckpt_name`` will be loaded."
+            f"For model, the default pattern is {MODEL_NAME_PATTERN}. For optimizer,"
+            f"the default pattern is {OPTIM_NAME_PATTERN}"),
+    )
+    parser.add_argument(
+        "--ckpt_name",
+        type=str,
+        default="",
+        help=(
+            f"The name pattern of the XLA FSDP checkpoint files to be consolidated. "
+            f"Files matching the pattern ``ckpt_dir + ckpt_name`` will be loaded."
+            f"For model, the default pattern is {MODEL_NAME_PATTERN}. For optimizer,"
+            f"the default pattern is {OPTIM_NAME_PATTERN}"),
+    )
+    parser.add_argument(
+        "--ckpt_type",
+        type=str,
```

Comment:
May be we could add a `choices` option for `ckpt_type`.

Reply:
done